README.md

```diff
@@ -52,10 +52,183 @@ The setup is done in 3 primary steps.  You will download the code, setup the app
+![](doc/source/images/wmlm-4.png)
+
+Once the training and evaluation is done, you can choose the best performed one and click `Save`.
+![](doc/source/images/wmlm-5.png)
+
+
+###8. Add Feedback data and new evaluations to the continuiously learning model
+
+Once the Watson Machine Learning Model is saved, select the `Evaluation` tab. First we need to configure the performance monitoring. 
+![](doc/source/images/wmlm-6.png)
+* Select the `Configure Performance Monitoring` link
+![](doc/source/images/wmlm-7.png)
+* Add the spark service you added to watson studio project
+* Choose `areaUnderPR` (performance metric of the model) and select the threshold as 0.8. This means if the performance is under 0.8, the model needs to be retrained using all the source data and new data and hence continuous learning.
+* Use `500` as record count and click `Save`.
+* Add the connection by selecting `Select Feedback Reference Data` and select the DB2 connection that you have created in the steps earlier.
+![](doc/source/images/feedback-data-2.png)
+
+* Once thats done, now you can add data using `+ Feedback Data`
+![](doc/source/images/add-feedback-data.png)
+
+* Once the feedback data is loaded, select `New Evaluation` to evaluate the uploaded feedback data. You can unzip the provided data [Chicago building inspection data by month 2017](building_inspection_data_by_month_2017.zip) in the repo and use that monthly inspection data as feedback data.
+![](doc/source/images/new-evaluation.png)
+
+* When the evaluation is completed we can see where the threshold value lies for this new feedback data.
+![](doc/source/images/evaluation_completion.png)
+
+* You can also see list of evaluations that has been completed and see how the model has been continuously learning
+![](doc/source/images/evaluation_completion-1.png)
+* You can upload new feedback data repeatedly from the provided data [Chicago building inspection data by month 2017](building_inspection_data_by_month_2017.zip)  so that the model cointinuously learns.
+
+###9. Deploy the model to expose it through an API
+
+* Select the model, and select `Deployments` tab. click `+ Add Deployment` to add a new deployment,
+![](doc/source/images/deployment-1.png)
+* Provide a name and choose `Web Service` as deployment type.
+![](doc/source/images/deployment-2.png)
+* Now the model is exposed through and API. If you select `Impelementation` tab you can see differnt examples on how to use the newly created API.
+![](doc/source/images/deployment-3.png)
+
+###9. Test the model
+
+To test the API, either you can use `curl` commands or through programming to access the API. You can also go to the `Test` tab and provide new set of data to evaluate the inspection status.
```

Comment:
"You can access and test the API programmatically, or use `curl` commands. "
"... and provide a new set ..."